reference/5.1/Microsoft.PowerShell.Utility/Write-Warning.md

```diff
@@ -21,58 +21,69 @@ Write-Warning [-Message] <String> [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Write-Warning** cmdlet writes a warning message to the Windows PowerShell host.
-The response to the warning depends on the value of the user's $WarningPreference variable and the use of the *WarningAction* common parameter.
+
+The `Write-Warning` cmdlet writes a warning message to the PowerShell host. The response to the
+warning depends on the value of the user's $WarningPreference variable and the use of the
+**WarningAction** common parameter.
 
 ## EXAMPLES
 
 ### Example 1: Write a warning message
 
-```
-PS C:\> Write-Warning "This is only a test warning."
-```
-
 This command displays the message "WARNING: This is only a test warning."
 
+```powershell
+Write-Warning "This is only a test warning."
+```
+
 ### Example 2: Pass a string to Write-Warning
 
-```
-PS C:\> $w = "This is only a test warning."
-PS C:\> $w | Write-Warning
-```
+This command shows that you can use a pipeline operator (`|`) to send a string to `Write-Warning`.
+You can save the string in a variable, as shown in this command, or pipe the string directly to
+`Write-Warning`.
 
-This command shows that you can use a pipeline operator (|) to send a string to **Write-Warning**.
-You can save the string in a variable, as shown in this command, or pipe the string directly to **Write-Warning**.
+```powershell
+$w = "This is only a test warning."
+$w | Write-Warning
+```
 
 ### Example 3: Set the $WarningPreference variable and write a warning
 
-```
-PS C:\> $warningpreference
-Continue PS C:\> Write-Warning "This is only a test warning."
-This is only a test warning. PS C:\> $warningpreference = "SilentlyContinue"
+This example shows the effect of the value of the `$WarningPreference` variable on a `Write-Warning`
+command.
+
+```powershell
+PS C:\> $WarningPreference
+Continue
+PS C:\> Write-Warning "This is only a test warning."
+This is only a test warning. PS C:\> $WarningPreference = "SilentlyContinue"
 PS C:\> Write-Warning "This is only a test warning."
-PS C:\>
-PS C:\> $warningpreference = "Stop"
+PS C:\> $WarningPreference = "Stop"
 PS C:\> Write-Warning "This is only a test warning."
 WARNING: This is only a test message.
 Write-Warning : Command execution stopped because the shell variable "WarningPreference" is set to Stop.
 At line:1 char:14
      + Write-Warning <<<<  "This is only a test message."
 ```
 
-This example shows the effect of the value of the $WarningPreference variable on a **Write-Warning** command.
-
-The first command displays the default value of the $WarningPreference variable, which is Continue.
-As a result, when you write a warning, the warning message is displayed and execution continues.
+The first command displays the default value of the `$WarningPreference` variable, which is
+Continue. As a result, when you write a warning, the warning message is displayed and execution
```

Comment:
Should `Continue` be formatted since it's a value?

Reply:
fixed